Google.PowerShell/Common/GcloudCmdlet.cs

```diff
@@ -87,9 +88,33 @@ protected string GetFullPath(string filePath)
                 return filePath;
             }
 
-            // For relative files, qualify them based on where the PowerShell session is.
-            string currentDirectory = SessionState.Path.CurrentLocation.Path;
-            return Path.Combine(currentDirectory, filePath);
+            ProviderInfo provider = null;
+            string[] result;
+
+            // resolve filepath and get the first one
```

Comment:
Comments should always be complete sentences. So begin with a capital letter and end with a period.